core/src/main/java/nl/nn/adapterframework/pipes/XmlWellFormedChecker.java

```diff
@@ -38,28 +42,35 @@
  * @author  Peter Leeuwenburgh
  * @since	4.4.5
  */
-
 public class XmlWellFormedChecker extends FixedForwardPipe {
 	private String root = null;
 
+	@Override
 	public void configure() throws ConfigurationException {
 		super.configure();
 		registerEvent(AbstractXmlValidator.XML_VALIDATOR_VALID_MONITOR_EVENT);
 		registerEvent(AbstractXmlValidator.XML_VALIDATOR_PARSER_ERROR_MONITOR_EVENT);
 	}
 
 
-	public PipeRunResult doPipe(Object input, IPipeLineSession session) {
-		if (XmlUtils.isWellFormed(input.toString(), getRoot())) {
+	@Override
+	public PipeRunResult doPipe(Message message, IPipeLineSession session) throws PipeRunException {
+		String input;
+		try {
+			input = message.asString();
+		} catch (IOException e) {
+			throw new PipeRunException(this, getLogPrefix(session)+"cannot open stream", e);
+		}
+		if (XmlUtils.isWellFormed(input, getRoot())) {
```

Comment:
kan dit geen Message als input aan, in de onderliggende methode wordt een saxsource gemaakt

Reply:
Ja, maar niet een gewone Source, maar een SAXSource, was er aan begonnen en ben er na een zucht mee opgehouden.
Andere keer.